incubator/kafka/README.md

```diff
@@ -175,3 +187,32 @@ such port at a time, setting the range at every Kafka pod is a reasonably safe c
 * There must not exist a service called `kafka` in the same namespace
 
 [brokerconfigs]: https://kafka.apache.org/documentation/#brokerconfigs
+
+## Prometheus Stats
+
+### Prometheus vs Prometheus Operator
+
+Prometheus is default option of this chart b/c the Prometheus Operator is part of another repo: 
+
+https://github.com/coreos/prometheus-operator/tree/master/helm 
+
+The Prometheus Operator gives additional functionality like automagically updating Prometheus and Alert Manager configuration.  These are things you will
+need to manually do if you are running Prometheus without the Operator.  For more details about the Prometheus Operator:
+
+https://coreos.com/blog/the-prometheus-operator.html
+
+### JMX Exporter
+
+https://github.com/prometheus/jmx_exporter Almost all the stats on Kafka come from JMX.  There are some limited complimentary stats that come from
+Kafka Exporter.  JMX Exporter is a generic exporter not only intended for Kafka.  When using with Kafka you get way more stats than one could want,
+white list them all or connect to JMX port (not Prometheus JMX Port used for scraping), also can see raw stats that way.
+
+JMX performs some transforms but without configuration metrics are not in a Prometheus format, things like broker names and topic names become part of
```

Comment:
To accommodate compatibility with the Prometheus collector, this chart performs transformations on top of the raw JMX metrics. For example, broker names and topics names are incorporated into the metric name instead of becoming a label. If you are curious to learn more about any default transformations to the chart metrics, please have [reference the configmap template](https://github.com/kubernetes/charts/blob/master/incubator/kafka/templates/jmx-configmap.yaml).